src/plugins/condition/components/Criterion.vue

```diff
@@ -15,39 +15,51 @@
                 </option>
             </select>
         </span>
-        <span class="controls">
-            <select v-model="criterion.metadata">
+        <span v-if="criterion.telemetry"
+              class="controls"
+        >
+            <select v-model="criterion.metadata"
+                    @change="updateOperations"
+            >
                 <option value="">- Select Field -</option>
-                <option v-for="option in telemetryMetadata"
+                <option v-for="option in telemetryMetadataOptions"
                         :key="option.key"
                         :value="option.key"
                 >
                     {{ option.name }}
                 </option>
             </select>
         </span>
-        <span class="controls">
+        <span v-if="criterion.telemetry && criterion.metadata"
+              class="controls"
+        >
             <select v-model="criterion.operation"
                     @change="updateOperationInputVisibility"
             >
                 <option value="">- Select Comparison -</option>
-                <option v-for="option in operations"
+                <option v-for="option in filteredOps"
                         :key="option.name"
                         :value="option.name"
                 >
                     {{ option.text }}
                 </option>
             </select>
-            <input v-if="isInputOperation"
-                   v-model="criterion.input"
+            <input v-if="inputCount > 0"
+                   v-model="criterion.input[0]"
+                   class="t-condition-name-input"
+                   type="text"
+                   @blur="persist"
+            >
+            <span v-if="inputCount === 2">and</span>
+            <input v-if="inputCount === 2"
+                   v-model="criterion.input[1]"
```

Comment:
Having multiple input elements like this is less than ideal and not scalable. A better option could be to use a v-for on the inputCount.
Something like this perhaps?
```suggestion
                   <div v-if="inputCount"
            >
                <span v-for="(item, index) in Array, inputCount"
                      :key="index"
                >
                    <input v-model="criterion.input[index]"
                           class="t-condition-name-input"
                           type="text"
                           @blur="persist"
                    >
                    <span v-if="index < inputCount-1">and</span>
                </span>
            </div>
```